src/graphql/mutations/login-user.ts

```diff
@@ -1,8 +1,10 @@
 import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
 
+import { uri } from '../uri';
+
 export const loginUser = async (email: string, password: string) => {
   const client = new ApolloClient({
-    uri: 'https://tq-template-server-sample.herokuapp.com/graphql',
+    uri: uri,
```

Comment:
aqui a chave (campo da esquerda) tem o mesmo nome que a variável (direita), dai pode omitir e ficar só
```suggestion
    uri,
```